int32 index   #index of object to be removed
---

